tools/main.c

```diff
@@ -69,8 +69,15 @@ main (int   argc,
         showVersion (argv[0]);
         exit (0);
     }
+
     if (opts.verbose)
         log_set_level(log_level_verbose);
```

Comment:
I believe so. I decoupled it purposefully because they go to a different standard stream anyways. Everything that's printed by verbose goes to stderr and everything that's affected by quiet is what goes to stdout.